ldms/src/ldmsd/ldmsd_config.c

```diff
@@ -89,7 +89,7 @@ LIST_HEAD(host_list_s, hostspec) host_list;
 LIST_HEAD(ldmsd_store_policy_list, ldmsd_store_policy) sp_list;
 pthread_mutex_t sp_list_lock = PTHREAD_MUTEX_INITIALIZER;
 
-#define LDMSD_PLUGIN_LIBPATH_MAX	1024
+#define LDMSD_PLUGIN_LIBPATH_MAX	LDMS_PLUGIN_LIBPATH_MAX
```

Comment:
Maybe just PATH_MAX from limits.h?

Reply:
no, path_max is unportably small sometimes (256 has been seen in the wild).
In any case, this is just an alias of existing code, not a change in behavior.